corehq/blobs/migrate_metadata.py

```diff
@@ -112,7 +118,7 @@ def _do_migration(self, doc):
                         ) VALUES (%s, %s, %s, %s, %s, %s, %s, CLOCK_TIMESTAMP())
                         ON CONFLICT (key) DO NOTHING
                     """, params=[
-                        (UNKNOWN_DOMAIN if obj.domain is None else obj.domain),
+                        domain,
                         type_code,
                         doc["_id"],
```

Comment:
Changed this in the rebase. It had mistakenly been changed to `obj._id`, which was wrong.